### PR TITLE
Changed the screenshot in README.md to be more friendly to new users

### PR DESCRIPTION
**Pull request by tomhiatt**
This was an issue I encountered because I don't really know what SSH is and it wasn't working for me in Rstudio. I'm guessing other new users like me will have more success with the HTTPS option.

BTW, this project is a great idea and something I fully welcome! Thank you!

PPS I'm pretty sure I messed something up with line wrapping, but if I'm correct, messing up is on par with the purpose of this repository.

### CODE REVIEW

**Comment by jonocarroll** [comment]
Thank you, and welcome!

I'm going to add some review comments so you can see how they would look, but so far so good!

File: man/figures/code.png
Comment (jonocarroll) [inline man/figures/code.png:1]: Now for the meat of the PR: I entirely overlooked using HTTPS instead of SSH and it's clearly a blind-spot I overlooked, so thank you for adding this! 

I think it might be helpful to include both options rather than _just_ the HTTPS one. It would also be good to resize the image a bit smaller. What do you think?

**Review by jonocarroll — CHANGES_REQUESTED** [review]
I've added some comments in line so that you can have a go at the process, but this PR overall looks good, thank you!

If you are okay with continuing, please have a look at the review comments.

File: README.md
Comment (tomhiatt) [reply README.md:7]: Yes, I should say I didn't mean to change any of the line wrapping, but when I opened it in R studio it forced me to make some decision on how it should change. Whatever decision I made seemed to automatically change the wrapping.

File: README.md
Comment (tomhiatt) [reply README.md:31]: Okay that's good to know. Again, this was an autochange...

File: README.md
Comment (tomhiatt) [reply README.md:41]: Yes, all of these changes were done with Rstudio automatic something. So this raises two questions.
1. Is there a way to avoid this when forking and cloning to Rstudio? 
2. I suppose it's a good idea to check each change before committing. What's the method you use to do that in Rstudio?

File: man/figures/code.png
Comment (tomhiatt) [reply man/figures/code.png:1]: Resizing sounds good. So in this case, would I do that and re-submit the PR? Or would you do it? How does that normally work?

As for keeping SSH, fine but it would need a little explanation or a link to more info I would think. The way you have it now is nice and simple.

**Comment by jonocarroll** [comment]
This is great - now we're both learning!

At the top of the Rstudio editor pane there's a cog; if you click that the top option is 'Use Visual Editor' which I recommend you disable, since it messes with the original file in all of the above ways. Now, that does mean that you need to work with the plaintext markdown file, but I strongly recommend this.

In Tools > Global Options > R Markdown > Visual there's a checkbox for 'Use visual editor by default' and I recommend disabling that, too, if you plan to make changes to other people's files.

The visual editor can be really helpful for editing your own files, but clearly it makes some opinionated changes to existing files, leading to a lot of unrelated changes in your pull request(s), which should be avoided. It's great that you encountered it here, though!

For reviewing your changes prior to making the pull request, I typically use the 'Diff' button in the git pane - this will show you what changes you're staging for the next commit, and in this case you'd see all of these changes before committing them. I always look through the diffs of the changed files to ensure that what I'm changing is what I intend. You can actually discard some of the changes by chunks from that screen, though I find that finicky.

Now, in terms of the changes here, my suggestion is to grab a copy of the README.md from the original repository (navigate to the file and click 'Raw' at the top to get the plaintext version) and copy that into your local folder, essentially 'resetting' the file. You can also just burn down (delete) what you have and start over; one of the great things about git. If you delete your fork and start over, you can create an entirely new pull request, but it might be worth working through the solution with the one you have.

You could commit the 'original' version of the file, or edit it and then do so. Once you push that commit to your fork it will update this PR, since this reflects the difference between the original repo and your fork. So that you have something I can merge, I'd say add the additional HTML-method image below the SSH-method one I had (resize yours if you like) and I'll take care of the remaining instructions - if you selected (or do select) 'Allow maintainers to edit' checkbox in this PR then I'll actually be able to push some changes to _your_ fork as well, which will update this PR.

Do let me know if you have any questions about any of that - I appreciate that it might look like this 'went wrong' but it did so in a really useful (in terms of learning experience) way, and I think it's fantastic.

**Comment by tomhiatt** [comment]
Okay I think I'm getting this! My PR is now much cleaned up (although somehow a diff in the link at the bottom still snuck through.) Let me know if there's still something missing.

Thanks for the detailed tutelage. So helpful!

**Comment by jonocarroll** [comment]
Looks Good To Me (LGTM)!

The change in that last line is a whitespace change - difficult to spot for sure. On this page on GitHub click 'Files Changed' at the top then the gear icon and you can turn on/off viewing whitespace changes. It's a good idea to make sure there aren't too many of these, but as the owner of the repository I don't usually mind if some of these sneak through - they're inconsequential.

I'm going to merge this, after which you're free to either delete your repository (since it will be identical to mine as you used your `main` branch as well) or keep it, but it won't stay in sync with any changes made here. Pushes to your main branch will no longer be reflected here, but the commits made here will be 'pulled in' to this repository and will become part of its history, so you don't need to worry about them. I should also mention, I'm going to do a regular merge in which case all 3 of the commits you made will be pulled in (the odd formatting, the fixing, and the re-edit) but there _is_ an option to 'squash' merge, which would combine all of your changes into a single commit in this repository's history. That's an option to keep in mind if you make many small changes and don't need them to stay separate in the resulting history.

Let me know if you have any other questions.

**Comment by jonocarroll** [comment]
Congrats on a successful pull request!

**Comment by tomhiatt** [comment]
Thank you! This was a very useful experience.

**Pull request merged**
